core/domain/exp_services.py

```diff
@@ -1622,6 +1623,43 @@ def get_next_page_of_all_non_private_commits(
     ) for entry in results], new_urlsafe_start_cursor, more)
 
 
+def get_image_filenames_from_exploration(exploration):
+    """ Get the image filenames from the exploration.
+
+    Args:
+        exploration: object. The exploration itself.
```

Comment:
"object" is not a suitable type here. You can be more specific.

Reply:
Done!